README.rst

```diff
@@ -107,11 +256,6 @@ Configuration
 Operations
 ==========
 
-Bootstrapping Consumers
------------------------
-
-.. todo:: Rewrite this as part of the tutorial using pgbench after the replication worker is done.
-
 Upgrades
 --------
 
```

Comment:
The rest of the README contains some stuff I leftover.  I have questions on a lot of it, so if anyone (@tkaemming) could elaborate on them I can try to fix it up.